docs/Examples.md

```diff
@@ -18,6 +18,71 @@ Delete Istio's envoy filter for translating `global` to `svc.cluster.local` at i
 
 ## Example Installations & Demos
 
+### Production Deployment
+
+![](Admiral_Diagram.png)
+
+
+In order to run admiral in production environment, we will have two types of clusters:
+- one main cluster where admiral lives
+- other remote clusters where admiral watches and monitors.
+
+The requirements are different for the two types:
+- admiral namespace will exist in main cluster
+- admiral-sync namespace will exist in remote clusters that admiral watches and monitors.
+
+1\. Set necessary environment variables  
+
+```bash
+# Set main cluster env variable
+export MAIN_CLUSTER=<path_to_kubeconfig_for_main_cluster>
+
+# Set remote cluster env variable
+export REMOTE_CLUSTER=<path_to_kubeconfig_for_remote_cluster>
+```
+
+2\. Install/Run Admiral in the main cluster
+
+```bash
+#Download and extract admiral
+
+wget https://github.com/istio-ecosystem/admiral/releases/download/v1.0/admiral-install-v1.0.tar.gz
+tar xvf admiral-install-v1.0.tar.gz
+
+export ADMIRAL_HOME=./admiral-install-v1.0
+```
+
+```bash
+#Install admiral
+export KUBECONFIG=$MAIN_CLUSTER
+$ADMIRAL_HOME/scripts/install_admiral.sh $ADMIRAL_HOME
+
+```
+
+3\. Add main cluster to Admiral's watcher
+
+Since there are most likely other contents living in the same cluster where admiral lives, admiral need to watch the cluster it's currently living in as well. This step can be skipped if Admiral has a dedicated cluster for Admiral only.
```

Comment:
```suggestion
Since there are most likely other workloads running in the same cluster where admiral lives, admiral needs to watch the cluster it's currently living in as well. This step can be skipped if Admiral runs in a dedicated cluster.
```